CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 ## 0.12.0-dev
   **BETA**
-
   Important: Do not use BETA features for production workflows.
-  
+  - [#372](https://github.com/Datatamer/tamr-client/issues/372) TC:Design for unified datasets
```

Comment:
I think this should go at the bottom of this section, below 
```
- Support for streaming records from a dataset via `tc.record.stream`
```
so that these are in chronological order.

Reply:
It was my assumption that they should be in chronological order with most recent at top.

Reply:
Historically, we haven't cared about order for changes that are part of the same release since those changes become user-facing at the same time.